### ISSUE TITLE
Step 5 on the page - shows multiple steps in small video - hard to follow

### ISSUE THREAD

**Issue by DS2019** [MicrosoftDocs/composer-docs#40]
In Step 5 (In this step you will need to enter some values in the different forms to generate your skills manifest.) there is a small video that very quickly shows a number of steps. It is hard to follow as it goes so fast. It would be much more helpful if I could pause it as I'm doing the steps for the first time or instead of a video, have the steps laid out in text and pictures.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 2a631930-1f27-93a4-7d70-79cdc6ee3bb3
* Version Independent ID: 8506ad08-a04a-31d6-46a0-24c4654c8705
* Content: [Export and connect to skills - Bot Composer](https://docs.microsoft.com/en-us/composer/how-to-connect-to-a-skill#create-azure-registration-resources)
* Content Source: [composer-docs/how-to-connect-to-a-skill.md](https://github.com/MicrosoftDocs/composer-docs-pr/blob/live/composer-docs/how-to-connect-to-a-skill.md)
* Service: **bot-composer**
* GitHub Login: @zxyanliu
* Microsoft Alias: **kamrani**

**Comment by zxyanliu** [comment]
Hi @DS2019 
Thanks for your feedback. Please be aware that connecting to a skill in Composer is a technical process that involves many steps such as setting up Composer and configuring Azure resources, and a high level of technical proficiency will be necessary to execute this process.
But I like your feedback that using a video that involves multiple steps may increase learning cost to our customers. 
The product will have a 1.2 release in the upcoming week, and this topic will be updated since the user experience will be changed. We will address this issue you mention by then. 
Will keep you posted on this. 

Thanks.

**Comment by zxyanliu** [comment]
Closing this issue for now and you are more than welcome to reopen it later.
